cypress/.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  env: {
+    node: true
+  },
+  extends: ['plugin:cypress/recommended', 'eslint:recommended'],
+  rules: {
+    quotes: [
+      'error',
+      'single',
+      {
+        allowTemplateLiterals: true,
+      },
+    ],
+    'no-trailing-spaces': 'error',
+    semi: [2, 'always'],
+  }
+};
```

Comment:
,eslintrc.js file I guess should be int the root of the project like cypress.config.js, etc. Like other configuration files.